osmose_config.py

```diff
@@ -1128,9 +1131,11 @@ class gen(default_country):
 south_africa_province("north_west", 349519)
 south_africa_province("northern_cape", 86720)
 south_africa_province("western_cape", 80501)
+south_africa_province("prince_edward_islands", 52947809)
```

Comment:
This relation doesn't exist in OSM.

Reply:
Hum, it is a way id. There is no relation.